src/components/EditDonationModal/EditDonationModal.graphql

```diff
@@ -36,6 +36,7 @@ query GetDesignationAccounts($accountListId: ID!) {
       id
       name
       active
+      designationNumber
```

Comment:
Do we also need to add the `EditDonationModalDonation` to the EditDonationModalDonation Fragment as well?

Reply:
@dr-bizz do you think it needs to be there? When mutating it just uses the `id` of the designation number, not the `designationNumber`

Reply:
I was thinking about when you update the donation and if it doesn't return a `designationNumber`, and there isn't a `name`, it will show blank.

Reply:
Ok I did that